nightly-main/windows/1809/Dockerfile

```diff
@@ -129,7 +129,7 @@ RUN Write-Host -NoNewLine ('Downloading {0} ... ' -f ${env:VSB});
 # Install Swift toolchain.
 ARG SWIFT_RELEASE_METADATA=https://download.swift.org/development/windows10/latest-build.json
 RUN $Release = curl.exe -sL ${env:SWIFT_RELEASE_METADATA} | ConvertFrom-JSON;   \
-    $SWIFT_URL = "\"https://download.swift.org/development/windows10/$($Release.dir)/$($Release.download)\""; \
+    $SWIFT_URL = "\"https://download.swift.org/development/windows10/swift-DEVELOPMENT-SNAPSHOT-2025-03-10-a/swift-DEVELOPMENT-SNAPSHOT-2025-03-10-a-windows10.exe\""; \
```

Comment:
suggestion: Can we comment out the original lines, with an indication it should be reverted once https://github.com/swiftlang/swift/issues/80356 is fixed?